.travis.yml

```diff
@@ -2,7 +2,10 @@ language: csharp
 
 solution: Castle.Core.sln
 
-mono: 4.6.1
+install:
+    - # Automatic package restore disabled. Packages will be restored by the build script.
```

Comment:
This is kind of hackish. A simple one-liner `install:` will be ignored by Travis, so we need either an empty line, or perhaps some kind of `echo` informational message.